_data/images.yml

```diff
@@ -7,3 +7,9 @@ new:
     name: 34C3 logo
     author: Gregor Sedlag
     origin: https://engelsystem.de/img/34c3_logo.png
+
+34c3-press:
+    url: 34c3-press.png
+    license:
```

Comment:
Please add at least your name to this, so people know where we got this from. The new entry also needs an author.